.github/workflows/playwright.yml

```
@@ -36,27 +20,47 @@ jobs:
           cache: 'npm'
 
       - name: Install Node dependencies
-        if: steps.cache-node-modules.outputs.cache-hit != 'true'
-        run: npm install --no-optional --no-audit
+        run: npm ci
 
-      - name: Load wp-env
-        run: npm run env:start --update
+      - name: Build Assets
+        run: FORCE_REDUCED_MOTION=true npm run build
 
-      - name: Fix permissions # We need to figure this out https://github.com/WordPress/gutenberg/issues/22515#issuecomment-1308346256
+      - name: blocks.ini setup
+        run: |
+          echo -e 'woocommerce_blocks_phase = 3\nwoocommerce_blocks_env = tests' > blocks.ini
+      - name: Get Composer Cache Directory
+        id: composer-cache
         run: |
-          WP_ENV_DIR=$(npm run wp-env install-path --silent 2>&1 | head -1)
-          cd $WP_ENV_DIR
-          mkdir -p tests-WordPress/wp-content/languages tests-WordPress/wp-content/upgrade
-          chmod -R 767 tests-WordPress/wp-content/languages tests-WordPress/wp-content/upgrade
-          cd -
+          echo "dir=$(composer config cache-files-dir)" >> $GITHUB_OUTPUT
+      - uses: actions/cache@v3
+        with:
+          path: ${{ steps.composer-cache.outputs.dir }}
+          key: ${{ runner.os }}-composer-${{ hashFiles('**/composer.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-composer-
```

What about if we simplify this part:

```suggestion
          restore-keys: ${{ runner.os }}-composer-
```